src/components/Modals/NewCourse/NewCourse.vue

```diff
@@ -125,13 +116,8 @@ export default Vue.extend({
     };
   },
   computed: {
-    text(): string {
-      return 'Search Course Roster';
-    },
```

Comment:
What's the benefit of not having this text be a computed property?

Reply:
It just bloats the component with computed properties. Usually it's only needed if you need some translations or conditional text